services/slam/builtin/cartographer_int_test.go

```diff
@@ -71,13 +71,16 @@ func TestCartographerIntegration(t *testing.T) {
 
 	t.Log("Testing online mode")
 
+	mapRate := 1
```

Comment:
Added the map rate flag to ensure multiple maps get saved, even if we are not using them in the tests. This ensures  a map is made before the end of a run, so if the maps do not get removed we should be able to see that.